x-pack/plugins/security_solution/public/management/cypress/e2e/sentinelone/isolate.cy.ts

```diff
@@ -0,0 +1,150 @@
+      // `internal/kibana/settings` is not accessible in serverless
+      '@brokenInServerless',
+    ],
+    env: {
+      ftrConfig: {
+        kbnServerArgs: [
+          `--xpack.securitySolution.enableExperimental=${JSON.stringify([
+            'sentinelOneManualHostActionsEnabled',
+          ])}`,
+          `--xpack.stack_connectors.enableExperimental=${JSON.stringify([
+            'sentinelOneConnectorOn',
+          ])}`,
+          `--xpack.actions.preconfigured=${JSON.stringify({
+            'preconfigured-sentinelone': {
+              name: 'preconfigured-sentinelone',
+              actionTypeId: '.sentinelone',
+              config: {
+                url: process.env.SENTINELONE_URL,
+              },
+              secrets: {
+                token: process.env.SENTINELONE_TOKEN,
+              },
+            },
+          })}`,
+        ],
+      },
+    },
+  },
+  () => {
+    before(() => {
+      cy.dataSession(
+        'SENTINELONE_HOST', // data name
+        () => cy.task('createSentinelOneHost', {}, { timeout: 100000000 }),
+        () => true
+      );
+      cy.dataSession('SENTINELONE_POLICY', () => cy.task('createSentinelOneAgentPolicy'), true);
+    });
+
+    after(() => {
+      cy.dataSession('SENTINELONE_HOST').then((createdHost) => {
+        if (createdHost && !Cypress.env('isInteractive')) {
+          createdHost.destroy();
+        }
+      });
+      cy.dataSession('SENTINELONE_POLICY').then((agentPolicy) => {
+        if (agentPolicy) {
+          agentPolicy.stop();
+          cy.task('deleteFleetServerPolicy', agentPolicy.policyId);
+        }
+      });
+    });
+
+    beforeEach(() => {
+      login(ROLE.soc_manager);
+      disableExpandableFlyoutAdvancedSettings();
+    });
+
+    describe('From alerts', () => {
+      let ruleId: string;
+      let ruleName: string;
+
+      before(() => {
+        cy.dataSession('SENTINELONE_HOST').then((createdHost) =>
+          loadRule(
+            {
+              index: ['logs-sentinel_one.alert*'],
+              query: `host.name: ${createdHost.name} and observer.serial_number:*`,
+              from: 'now-3660s',
+            },
+            false
+          ).then((data) => {
+            ruleId = data.id;
+            ruleName = data.name;
+          })
+        );
+      });
+
+      after(() => {
+        if (ruleId) {
+          cleanupRule(ruleId);
+        }
+      });
+
+      it('should isolate and release host', () => {
+        toggleRuleOffAndOn(ruleName);
+        visitRuleAlerts(ruleName);
+        waitForAlertsToPopulate();
+
+        closeAllToasts();
+        openAlertDetailsView();
+
+        cy.getByTestSubj('isolate-host-action-item').click();
+        cy.getByTestSubj('hostIsolateConfirmButton').click();
+        cy.dataSession('SENTINELONE_HOST').then((createdHost) => {
+          cy.log('create', createdHost);
+          cy.contains(`Isolation on host ${createdHost.name} successfully submitted`);
```

Comment:
(optional) I see use of `cy.contains()` in several places. Might be better to now use it and instead narrow your assertions to specific elements in the UI (using (for example) `data-test-subj` values).